content/en/docs/reference/command-line-tools-reference/feature-gates/image-maximum-gc-age.md

```diff
@@ -6,8 +6,8 @@ _build:
   render: false
 
 stages:
-  - stage: alpha 
-    defaultValue: false
-    fromVersion: "1.29"   
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
```

Comment:
The changes to the `stages` block appear incorrect. Please refer to the documentation [here](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates) for further information. Additionally, I've provided some suggested changes below for your consideration. I hope these recommendations prove helpful.

```suggestion
  - stage: alpha 
    defaultValue: false
    fromVersion: "1.29"
    toVersion: "1.29"    
  - stage: beta
    defaultValue: true
    fromVersion: "1.30"
```

Reply:
updated!